components/admin_console/license_settings/license_settings.tsx

```diff
@@ -415,8 +415,9 @@ export default class LicenseSettings extends React.PureComponent<Props, State> {
                 <div className='admin-console__wrapper'>
                     <div className='admin-console__content'>
                         <div className='admin-console__banner_section'>
-                            {(license.IsLicensed !== 'true') && (this.props.prevTrialLicense?.IsLicensed !== 'true') &&
-                                free30DayTrial(
+                            {this.props.enterpriseReady && (license.IsLicensed !== 'true') &&
```

Comment:
Adding the enterpriseReady condition we address ticket https://mattermost.atlassian.net/browse/MM-40169

Reply:
Awesome, thanks for incorporating this fix here 👍